docs/UserGuide.md

```diff
@@ -127,7 +128,11 @@ ___
 Adds a user defined anime into the anime list.
 Format: `add n/NAME [e/EPISODE] [s/STATUS] [g/GENRE] [g/GENRE] ...`
 
-Note: If multiple `n/NAME` is given in the command, the title of the added anime will be the last provided `NAME`
+Notes:
+* If multiple `n/NAME` is given in the command, the title of the added anime will be the last provided `NAME`
```

Comment:
should we specify it for all of the parameters except for genre? as this applies to everything but genre

Reply:
yah we should, but may make the the ug very wordy

Reply:
this behaviour occurs not just for the add command right, maybe Ill add this notes in the command format notes